src/Components/Topics/Vote.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import { patchVoteById } from '../API/Api'
+import Button from './Button'
+import styles from './Vote.module.css'
+
+function Vote(props) {
+    const [disableLikeButton, setDisableLikeButton] = useState(false)
+    const [disableUnLikeButton, setDisableUnLikeButton] = useState(true)
+    const [error, setError] = useState(false)
+
+    function VoteHandlerApi(voteCount) {
+        setDisableLikeButton((prevState) => !prevState)
+        if (voteCount === 1) {
+            props.setVotes((prevVote) => prevVote + 1)
+        } else {
+            props.setVotes((prevVote) => prevVote - 1)
+        }
+        patchVoteById(props.article.article_id, voteCount).catch((err) => {
+            setError(true)
+        })
+        setDisableUnLikeButton((prevState) => !prevState)
+    }
+
+    return (
+        <div className={styles.vote}>
+            <p>
+                <b>CLICK TO VOTE</b>
+            </p>
+            <Button
+                disabled={disableLikeButton ? 'disabled' : ''}
```

Comment:
> This **Boolean** attribute prevents the user from interacting with the button: it cannot be pressed or focused.
(MDN)

You're passing a string!

`disableLikeButton` is already a boolean so you should be able to just pass in that